templates/javascript-ethers/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "javascript-ethers-beacon-starter",
```

Comment:
Shouldn't we use a name based on the directory/project name?

Reply:
That is not that trivial actually, since the services repo is not build to edit the generated files (only to copy them as of now).

But I agree, I'll create a Jira ticket for this. For now I think it can stay like this.